src/stock-card/stock-card.service.spec.js

```diff
@@ -36,31 +37,11 @@ describe('stockCardService', function() {
     });
 
     describe('getStockCard', function() {
-
-        beforeEach(function() {
-            this.$httpBackend.when('GET', this.stockmanagementUrlFactory('/api/stockCards/' + this.stockCard.id))
-                .respond(200, this.stockCard);
-        });
-
         it('should return promise', function() {
             var result = this.stockCardService.getStockCard(this.stockCard.id);
-            this.$httpBackend.flush();
 
             expect(result.then).not.toBeUndefined();
         });
-
-        it('should resolve to stock card', function() {
```

Comment:
Why was this test removed? Shouldn't it be adapted to the new code with stockCardResource?

Reply:
the test was removed because the get functionality is tested already in the OpenlmisCachedResource code